src/app.js

```diff
@@ -1 +1,46 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+
+async function fetchCountries() {
+    try {
+        const response = await axios.get('https://restcountries.com/v3.1/all?fields=name,population,flags,continents');
+        const countryData = response.data;
+        countryData.sort((a, b) => a.population - b.population)
+
+        const countryList = document.getElementById('countriesList');
+        const listOutput = countryData.map((country) => {
+            const regionColor = getRegionColor(country.continents[0]);
+            countryList.innerHTML += `
+            <li><img src="${country.flags.png}" alt="country flag"> 
```

Comment:
Beetje nitpicky is het wel, maar in het country object is er voor iedere vlag ook een alt value.